tests/network/vmi_networking.go

```diff
@@ -778,9 +785,10 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 
 				Expect(verifyClientServerConnectivity(clientVMI, serverVMI, tcpPort, k8sv1.IPv6Protocol)).To(Succeed())
 			},
-				table.Entry("with a specific port number [IPv6]", []v1.Port{{Name: "http", Port: 8080}}, ""),
-				table.Entry("without a specific port number [IPv6]", []v1.Port{}, ""),
-				table.Entry("with custom CIDR [IPv6]", []v1.Port{}, "fd10:10:10::/120"),
+				table.Entry("with a specific port number [IPv6]", []v1.Port{{Name: "http", Port: 8080}}, 8080, ""),
+				table.Entry("with a specific port used by live migration", portsUsedByLiveMigration(), int(portsUsedByLiveMigration()[0].Port), ""),
```

Comment:
I think that if the test is against only one port out of the three, you can choose different indexes in the different test cases. for example use  `portsUsedByLiveMigration()[0]` for ipv4 and `portsUsedByLiveMigration()[0]` for ipv6. What do you think?

Reply:
I don't think this is a good thing to do. I'm already not sure if using the index is OK for readability. Using different indexes may be confusing for a reader.
I'm worried that it may imply that there is some sort of difference between the ports. But there isn't any.
IMO, using different ports out of the three won't test the rules any better, than using just one of them.